src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ConfigurationController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api
+{
+    [ApiController]
+    public class ConfigurationController
+    {
+        private readonly IConfiguration _configuration;
+        public ConfigurationController(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        // GET api/configuration/application-insights/
+        [HttpGet("api/configuration/application-insights")]
+        public string GetInsightsKey()
```

Comment:
Might wanna change this return type to `ActionResult<string>`, and then change the code in the method to return:

```
Ok(instrumentationKey)
```

if one is found in the configuration, or:

```
NotFound()
```

if no instrumentation key was found.

That way, you either get a 200 or a 404 back from the API rather than always a 200.